flang/test/Driver/frame-pointer-forwarding.f90

```diff
@@ -1,9 +1,12 @@
 ! Test that flang-new forwards -fno-omit-frame-pointer and -fomit-frame-pointer Flang frontend
-! RUN: %flang -fno-omit-frame-pointer --target=x86-none-none -fsyntax-only -### %s -o %t 2>&1  | FileCheck %s
-! CHECK: "-mframe-pointer=all"
+! RUN: %flang --target=aarch64-none-none -fsyntax-only -### %s -o %t 2>&1  | FileCheck %s --check-prefix=CHECK-NOVALUE
+! CHECK-NOVALUE: "-mframe-pointer=non-leaf"
```

Comment:
```suggestion
! CHECK-NOVALUE: "-fc1" "-mframe-pointer=non-leaf"
```

Same remark for other check lines.

Reply:
If we add this then it will be added as `"-fc1"{{.*}}"`.  Why is it necessary?

Reply:
See the comment at the top of the file:

> ! Test that flang-new forwards -fno-omit-frame-pointer and -fomit-frame-pointer Flang frontend

In order to identify that this is indeed checking the **frontend** driver invocation (as per the comment, that's the intention), you need to check for `-fc1`. That's something that was missing here to begin with, so strictly speaking it's tangential to your change. But since you are updating these CHECK lines anyway ...